files/about.html

```diff
@@ -72,8 +72,8 @@ <h2>Let's grow together!</h2>
         <h1 id = "who-we-are">Who We Are</h1>
     
         <p id = "who-we-are-content">
-        The Computer Science Society (CSS) Club is dedicated to promoting the STEM field,
-        specifically computer science, technology, and mathematics. We work to provide resources for any student, alumni,
+        The Computer Science Society (CSS) Club are dedicated to promoting the STEM field,
```

Comment:
I think we can keep it as "is", rather than change it to "are", as we are one entity. Other than that everything looks great. Feel free to merge afterwards

Reply:
@r0m3c ✅